index.html

```diff
@@ -19,14 +19,15 @@
   <div class="ellipse-one"></div>
   <div class="ellipse-two"></div>
 
-  <header class="header">
+  <header class="header" id="port-folio">
     <div class="logo">
       <a href="#" class="link" target=" blank">DU</a>
     </div>
 
-    <button type="button" class="hamburger">
+    <button type="button" class="hamburger" onclick="menuFunction()">
       <span class="bars"></span>
     </button>
+   
     <nav class="nav-bar">
```

Comment:
**Great job making your hamburger menu work as expected, but there are some issues that you still need to work on to go to the next project but you are almost there!** : 

- [ ]  The mobile menu should have margins as defined in the design. Kindly, stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to make the hamburger menu closely similar to the [original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 😸.
- [ ] Kindly add the blur background images as you can see in the screenshots below
- [ ] Kindly make sure that the mobile menu has a `height: 100vh;` 

**Current mobile menu**  |  **Original design**
:----------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/156154903-d7fff769-69f6-4642-aaa2-4cf9846897b5.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/156154910-9933a080-75cd-4e3f-be96-39fdab8caf7d.png)